benches/lib_benches.rs

```diff
@@ -32,17 +32,20 @@ bench_decimal_op!(add_pi, +, "3.1415926535897932384626433832");
 bench_decimal_op!(add_negative_pi, +, "-3.1415926535897932384626433832");
 
 #[bench]
-fn bench_sum_10m(b: &mut ::test::Bencher) {
-    fn sum_10m(values: &[Decimal]) -> Decimal {
+fn bench_sum_10k(b: &mut ::test::Bencher) {
```

Comment:
Looking at the two of these functions I'm thinking that `bench_add_10k` and `bench_sub_10k` may be better.

Reply:
Done, as I understand